lib/src/default_index/revset_engine.rs

```diff
@@ -133,18 +130,6 @@ where
         Box::new(self.iter_graph_impl())
     }
 
-    fn change_id_index(&self) -> Box<dyn ChangeIdIndex + 'index> {
-        // TODO: Create a persistent lookup from change id to commit ids.
```

Comment:
nit: this TODO is still valid, but we have another comment about that in `CommitGraphEntry`, so it should be okay to delete now.

(no action needed)

Reply:
I actually meant to keep the comment. I've added it back in readonly.rs